pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -600,7 +610,28 @@ func (exp *FlowExporter) findFlowType(conn flowexporter.Connection) uint8 {
 	return 0
 }
 
+func (exp *FlowExporter) fillEgressPolicyInfo(conn *flowexporter.Connection) {
+	egressPolicyName, err := exp.egressQuerier.GetEgress(conn.SourcePodNamespace, conn.SourcePodName)
+	if err != nil {
+		return
+	}
+	conn.EgressPolicyName = egressPolicyName
+	egressIP, err := exp.egressQuerier.GetEgressIPByName(conn.EgressPolicyName)
+	if err != nil {
+		klog.ErrorS(err, "error when try to get Egress IP by Egress Policy Name", "Egress Policy", conn.EgressPolicyName)
+	}
+	conn.EgressIP = egressIP
+	klog.V(4).InfoS("Fill Egress Policy Info for flow", "Egress Policy", conn.EgressPolicyName, "Egress IP", conn.EgressIP, "Source Pod Namespace", conn.SourcePodNamespace, "Source Pod Name", conn.SourcePodName)
+}
+
 func (exp *FlowExporter) exportConn(conn *flowexporter.Connection) error {
```

Comment:
as I pointed out in your go-ipfix PR, maybe we could skip exporting the connection at the Egress Node (if different from the Source Node), assuming we can easily identify it?

Reply:
Yes it's a better option, I will make changes.